src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -194,8 +194,12 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         "Show message reply context", s.hideReplyContext, true,
         "This setting will only affect how messages are shown. You can reply "
         "to a message regardless of this setting.");
-    layout.addCheckbox("Show message reply button", s.showReplyButton);
-    layout.addCheckbox("Show tab close button", s.showTabCloseButton);
+    layout.addCheckbox("Show message reply button", s.showReplyButton, false,
+                       "Show a reply button next to every chat message");
+    layout.addCheckbox(
+        "Show tab close button", s.showTabCloseButton, false,
+        "When disabled, the x to close a tab will be hidden.\nTabs can still "
+        "be closed by right-clicking or CTRL+SHIFT+W");
```

Comment:
This tooltip should mention the defaultness of this or preferably use HotkeyController::getDisplaySequence introduced in #4045:

https://github.com/Chatterino/chatterino2/pull/4045/files#diff-d39bf5472005a5ba68ffdb09e75289913c7f1dd9b0285fd6acd44c799ac6771dR77-R86